addons/website/static/src/components/fields/fields.js

```diff
@@ -114,3 +116,38 @@ FieldFaPrefix.template = 'website.FieldFaPrefix';
 fieldRegistry.add('page_url', FieldPageUrl);
 fieldRegistry.add('page_name', FieldPageName);
 fieldRegistry.add('fa_prefix', FieldFaPrefix);
+
+/**
+ * Displays 'Selection' field's values as images to select.
+ * Image src for each value can be added using the option 'images' on field XML.
+ */
+export class ImageRadioField extends Component {
+    setup() {
+        const selection = this.props.record.fields[this.props.name].selection;
+        this.values = selection.filter(item => {
+            return item[0] || item[1];
+        }).map((value, index) => {
+            return [...value, this.props.images && this.props.images[index] || ''];
+        });
+    }
+
+    /**
+     * @param {String} value
+     */
+    onSelectValue(value) {
+        this.props.update(value);
+    }
```

Comment:
Can you explain this a bit to the owl noob I am ? If you have `props` that are updated should not they be part of the `state` which triggers a re-render? Or do props also trigger a re-render?

Reply:
To my knowledge, `useState` was applied before on component `props` but since [this update](https://github.com/odoo/owl/commit/4c77132ae2bb84bd1ccf8c98c7fb3a56a8163b93) the `useState` is only applied on reactive ones. The `update()` here is a field related property used to update the value on the linked record, check if the value should be saved,...